Solutions/MimecastAudit/SolutionMetadata.json

```diff
@@ -0,0 +1,20 @@
+{
+    "publisherId": "azuresentinel",

```

Comment:
Does your partner center supports this solution id, if not change the solution id.